modules/core/src/test/java/org/apache/ignite/cache/RebalanceCancellationTest.java

```diff
@@ -107,87 +120,112 @@
         cleanPersistenceDir();
     }
 
-    //--
-
-    /** */
+    /**
+     * Non baseline node leaves cluster with only persistent caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeLeftOnOnlyPersistenceCluster() throws Exception {
         testRebalanceNoneBltNode(true, false, false);
     }
 
-    /** */
+    /**
+     * Non baseline node leaves cluster with only memory caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeLeftOnOnlyInMemoryCluster() throws Exception {
         testRebalanceNoneBltNode(false, false, false);
     }
 
-    /** */
+    /**
+     * Non baseline node leaves cluster with persistent and memory caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeLeftOnMixedCluster() throws Exception {
         testRebalanceNoneBltNode(true, true, false);
     }
 
-    //--
-
-    /** */
+    /**
+     * Non baseline node fails in cluster with only persistent caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeFailedOnOnlyPersistenceCluster() throws Exception {
         testRebalanceNoneBltNode(true, false, true);
     }
 
-    /** */
+    /**
+     * Non baseline node fails in cluster with only memory caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeFailedOnOnlyInMemoryCluster() throws Exception {
         testRebalanceNoneBltNode(false, false, true);
     }
 
-    /** */
+    /**
+     * Non baseline node fails in cluster with persistent and memory caches during rebalance.
+     */
     @Test
     public void testRebalanceNoneBltNodeFailedOnMixedCluster() throws Exception {
         testRebalanceNoneBltNode(true, true, true);
     }
 
-    //--
-
-    /** */
+    /**
+     * Filtered node leaves cluster with persistent region.
+     */
     @Test
+    @Ignore("https://ggsystems.atlassian.net/browse/GG-26764")
     public void testRebalanceFilteredNodeOnOnlyPersistenceCluster() throws Exception {
-        testRebalanceNoneBltNode(true, false, true);
+        testRebalanceFilteredNode(true, false);
     }
 
-    /** */
+    /**
+     * Filtered node leaves cluster with memory region.
```

Comment:
Need a description about the exception, for example "@throws Exception If failed".